src/Features/Core/Portable/Completion/CommonCompletionUtilities.cs

```diff
@@ -101,7 +101,9 @@ public static bool IsStartingNewWord(SourceText text, int characterPosition, Fun
 
             // TODO(cyrusn): Figure out a way to cancel this.
             var symbol = symbols[0];
-            var sections = await symbolDisplayService.ToDescriptionGroupsAsync(workspace, semanticModel, position, ImmutableArray.Create(symbol), cancellationToken).ConfigureAwait(false);
+            var sections = await symbolDisplayService.ToDescriptionGroupsAsync(
+                workspace, semanticModel, position, ImmutableArray.Create(symbol), 
+                ImmutableArray<SyntaxNode>.Empty, cancellationToken).ConfigureAwait(false);
```

Comment:
Can you remove hte "figure out a way to cancel this"?  :)  given that clearly found a way.

Reply:
Needs comment explaining that we do not want capture groups here (i.e. because those are for keywords of a query, and this is not the code that presents those).